plugins/azure/virtualmachines/computeGalleryRbac.js

```diff
@@ -0,0 +1,51 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure/');
+
+module.exports = {
+    title: 'Compute Gallery RBAC Sharing',
+    category: 'Virtual Machines',
+    domain: 'Compute',
+    description: 'Ensures that the Azure Compute Gallery is using RBAC only.',
+    more_info: 'As the Azure Compute Gallery, definition, and version are all resources, they can be shared using the built-in native Azure Roles-based Access Control (RBAC) roles. A direct shared gallery can\'t contain encrypted image versions. Community galleries can be used by anyone with an Azure subscription making the resource more vulnerable.',
+    recommended_action: 'Ensure that all Azure Compute Galleries are using RBAC only.',
+    link: 'https://learn.microsoft.com/en-us/azure/virtual-machines/azure-compute-gallery',
+    apis: ['computeGalleries:list'],
```

Comment:
```suggestion
    apis: ['computeGalleries:list'],
```
add realtime triggers